flow/scene_update_context.cc

```diff
@@ -72,14 +73,13 @@ void SceneUpdateContext::CreateFrame(scenic::EntityNode entity_node,
   // and possibly for its texture.
   // TODO(SCN-137): Need to be able to express the radii as vectors.
   scenic::ShapeNode shape_node(session());
-  scenic::RoundedRectangle shape(
-      session_,                                      // session
-      rrect.width(),                                 // width
-      rrect.height(),                                // height
-      rrect.radii(SkRRect::kUpperLeft_Corner).x(),   // top_left_radius
-      rrect.radii(SkRRect::kUpperRight_Corner).x(),  // top_right_radius
-      rrect.radii(SkRRect::kLowerRight_Corner).x(),  // bottom_right_radius
-      rrect.radii(SkRRect::kLowerLeft_Corner).x()    // bottom_left_radius
+  scenic::RoundedRectangle shape(session_,        // session
```

Comment:
This should just be a scenic::Rectangle

Reply:
If it's just a rectangle, why are we not using `scenic::Rectangle` here? Does `scenic::Rectangle` exist? It might be nice to write the answers as code comment just in case that someone will have similar questions by reading the code.

Reply:
yeah, my comment was meant to say I should just use scenic::Rectangle :) will include that in fixes

Reply:
Done